.bcr/metadata.template.json

```diff
@@ -0,0 +1,12 @@
+{
+  "homepage": "https://github.com/capnproto/capnproto",
+  "maintainers": [
+    {
+      "github": "kentonv",
+      "name": "Kenton Varda"
+    }
+  ],
```

Comment:
Module template; @kentonv, you are listed, and you may want to list others, I don't know. Anybody listed here will show up on the BCR module page. I stopped short of listing your email ;)